CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Export
+Service to export YourLoops data in selected formats, either csv, json or xls.
+
+## [1.3.2] - 2019-09-10
```

Comment:
Should we use our version number ?
1.3.2 is the tidepool version.

Reply:
Let's use the 1.3.2 for now.